### Geofrenzy GfDwellColor message
#
# A dwell message specifies a current entitlement in relation to the tracked
# location.
#
# The color dwell ... 
#
Header      header      # Standard ROS header
GfEntColor  entitlement # Dwell entitlement
